packages/configs/eslint-common-backend.json

```diff
@@ -0,0 +1,50 @@
+{
+    "overrides": [
+        {
+            "files": "./backend/**/*.ts",
+            "extends": [
+                "./eslint-common-node.json",
+                "plugin:node/recommended-module"
+            ],
+            "rules": {
+                "prefer-promise-reject-errors": "off",
```

Comment:
Hm... How difficult would it be to remove this line? As we implemented RD-571 (I know it was about unhandled rejections and not necessarily about using error throwing), maybe not that big? 🤔

Reply:
Checked and it looks like we can just remove it without further alignments - done.